### PR TITLE
refactor(PreviewWindow): move zoom buttons to window title

### LINKED ISSUE #146: In the single-page view, move the zoom buttons to the window title
From https://github.com/junrrein/pdfslicer/issues/86#issuecomment-514519115 (@amivaleo):

>(...) I suggest to remove the overlay zoom-buttons and replace them with a set a buttons like in eog.
>
>
>
>The only extra I'd like to suggest is to use a slider in the popover menu that appears if you click on the number in the middle.

### PR DESCRIPTION
**Pull request by miltolstoy**
Resolves #146.

### CODE REVIEW

File: src/application/previewheaderbar.cpp [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// PDF Slicer
+// Copyright (C) 2017-2020 Julián Unrrein
```
Comment (junrrein) [inline src/application/previewheaderbar.cpp:2]: For files you create, you are the copyright holder! Put your name here, and the current year. Same thing in previewheader.hpp.

Comment (junrrein) [reply src/application/previewheaderbar.cpp:2]: As for the code you implemented, it's just what was needed. Good job :+1:

File: src/application/CMakeLists.txt [hunk 2]
```diff
@@ -1,26 +1,4 @@
...
-	${CMAKE_CURRENT_SOURCE_DIR}/savefiledialog.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/savingrevealer.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/task.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/taskrunner.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/unsavedchangesdialog.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/view.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/welcomescreen.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/zoomlevel.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/zoomlevelwithactions.cpp)
+file (GLOB APPLICATION_SOURCES
```
[the remark this answers is not in the document]
Comment (miltolstoy) [reply src/application/CMakeLists.txt:1]: previewheaderbar.cpp was required to add to sources list. Just made it simpler.

File: src/application/previewheaderbar.cpp (hunk 1 above)
Comment (miltolstoy) [reply src/application/previewheaderbar.cpp:2]: > As for the code you implemented, it's just what was needed. Good job +1
My pleasure.

File: src/application/previewheaderbar.hpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#pragma once
+
+#include "appmenu.hpp"
```
Comment (junrrein) [inline src/application/previewheaderbar.hpp:19]: Is this `#include` needed?

Comment (miltolstoy) [reply src/application/previewheaderbar.hpp:19]: It provides Gtk::Button.
Changed to <gtkmm/button.h> to be more lucid.

**Comment by miltolstoy** [comment]
Moved to #195

**Pull request closed without merging**